HandleNHNotificationCallActivity/handler.ts

```diff
@@ -53,9 +55,20 @@ export const getCallNHServiceActivityHandler = (
       context.log.info(
         `${logPrefix}|${message.kind}|INSTALLATION_ID=${message.installationId}`
       );
+
+      const nhService = getNHLegacyService();
+
+      if (isNone(nhService)) {
+        context.log.warn(
+          `${logPrefix}|${message.kind}|INSTALLATION_ID=${message.installationId}|No NH `
+        );
+        return;
```

Comment:
never return `undefined`! I'm also concerned of why the compiler didn't complain...

Reply:
This is from the code I copied. I have to admit that I didn't check it..

Reply:
Done in [eeae731](https://github.com/pagopa/io-functions-pushnotifications/pull/4/commits/eeae73106a00e5552c744ef0e3771d9715cfd51d)